### PR TITLE
Card structure and draw data

### PR DESCRIPTION
**Pull request by ShaharShi**
**This branch include:**

- Cards Style
- Draw Cards after page refresh , as the number saved in LS.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  return `${user.name.first} ${user.name.last}`;
+}
+function getCountry(user) {
+  return user.location.country;
+}
+function getCity(user) {
+  return user.location.city;
+}
+function getAdress(user) {
+  return `${user.location.street.name} ${user.location.street.number}`;
```
Comment (galamo) [inline app.js:39]: good job
what about validations? user.location.street.name
what will happened if user is not OBJ and location is not OBJ ? think about that

File: app.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+}
+function getUserImgUrl(user) {
+  return user.picture.large;
+}
+
+async function init(numberOfUsers, cardsContainer) {
```
Comment (galamo) [inline app.js:45]: replace function init
with more declarative name

File: app.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  const mappedUsers = arrOfObjects.map((user) => {
+    return getMappedUser(user);
+  });
+  console.log(mappedUsers);
+
+  const cards = mappedUsers.map((user) => {
+    return getCardItem(user);
+  });
+
+  cardsContainer.append(...cards);
```
Comment (galamo) [inline app.js:70]: GOOD JOB

File: app.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    const { path } = settingObj;
+    const isFunction = typeof settingObj["fn"] === "function";
+    return {
+      ...mappedUser,
+      [key]: isFunction ? settingObj["fn"](user) : getValueFromPath(path, user),
+    };
+  }, {});
+}
+
+function getCardItem(user) {
```
Comment (galamo) [inline app.js:87]: nice

File: index.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    <div class="jumbotron">
+      <h1 class="ml-5">Users Data</h1>
+    </div>
+    <div class="input-group ml-3 col-xl-5 col-lg-6 col-md-8 col-sm-10 col-11">
+      <div class="input-group-prepend">
+        <label class="input-group-text" for="inputGroupSelect01"
+          >How many users do you want ?
+        </label>
+      </div>
+      <select class="custom-select" id="inputGroupSelect01">
```
Comment (galamo) [inline index.html:26]: generate this values from JS config please
\

File: app.js (hunk 1 above)
Comment (ShaharShi) [reply app.js:39]: So i did'nt understand why we changed `getValueFromPath()` function to all `getName(),getAdress()...` functions.
becouse `getValueFromPath() ` does the validation itself..
and if we want to do the validation through the `get...()` functions, we need to call the `getValueFromPath()` in every `get...() `function, am i right ?

**Pull request merged**